pdc_client/__init__.py

```diff
@@ -91,27 +91,71 @@ def read_config_file(server_alias):
 
 
 class PDCClient(object):
-    """BeanBag wrapper specialized for PDC access.
+    """
+    BeanBag wrapper specialized for PDC access.
 
     This class wraps general BeanBag.v1 objects, but provides easy-to-use
     interface that can use configuration files for specifying server
     connections. The authentication token is automatically retrieved (if
     needed).
     """
+
+    # Set a page_size which is used in _WrapperBeanBag
+    page_size = None
+
+    class _WrapperBeanBag(beanbag.BeanBag):
+        """
+        BeanBag wrapper for getting the constructor's page_size in used.
+
+        Re-implement the call function as in beanbag you can't call
+        super() function of parent class.
+        """
+
+        def call(self, path, *args, **kwargs):
+            """Make a GET, POST or generic request to a resource.
+
+               :Example:
+
+               >>> x = BeanBag("http://host/api")
+               >>> r = x()                                 # GET request
+               >>> r = x(p1='foo', p2=3)                   # GET request with parameters passed via query string
+               >>> r = x( {'a': 1, 'b': 2} )               # POST request
+               >>> r = x( "RANDOMIZE", {'a': 1, 'b': 2} )  # Custom HTTP verb with request body
+               >>> r = x( "OPTIONS", None )                # Custom HTTP verb with empty request body
+            """
+            if PDCClient.page_size is not None:
```

Comment:
`PDCClient.page_size` is static, you need `page_size` for given instance of `PDCClient`.

Reply:
Yes, I just want to use this static parameter. As I don't want to copy the __init__ function again.